azure-pipelines/e2e-assets/test-dll-port-template/portfile.cmake

```
@@ -1,14 +1,69 @@
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/bin")
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/lib")
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug")
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug/bin")
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug/lib")
 file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/include")
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/share")
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/share/${PORT}")
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/debug/test_dll.dll" DESTINATION "${CURRENT_PACKAGES_DIR}/debug/bin")
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/debug/test_dll.lib" DESTINATION "${CURRENT_PACKAGES_DIR}/debug/lib")
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/release/test_dll.dll" DESTINATION "${CURRENT_PACKAGES_DIR}/bin")
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/release/test_dll.lib" DESTINATION "${CURRENT_PACKAGES_DIR}/lib")
 file(COPY "${CMAKE_CURRENT_LIST_DIR}/test.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
+
+file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 file(TOUCH "${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright")
+
+if(release-only IN_LIST FEATURES AND NOT bad-release-only IN_LIST FEATURES)
```

```suggestion
if("release-only" IN_LIST FEATURES AND NOT "bad-release-only" IN_LIST FEATURES)
```

I'm not 100% sure that this is a problem (due to the hyphen), but it does make me very nervous about accidental variable expansion.

---

I'm 100% sure this isn't a problem due to IN_LIST but I'm making the change anyway.